hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -4136,9 +4169,69 @@ func reconcileVirtLauncherNetworkPolicy(policy *networkingv1.NetworkPolicy, hclu
 			},
 		},
 	}
-	policy.Spec.PodSelector = metav1.LabelSelector{
-		MatchLabels: map[string]string{
-			"kubevirt.io": "virt-launcher",
+	policy.Spec.Egress = []networkingv1.NetworkPolicyEgressRule{
+		{
+			To: []networkingv1.NetworkPolicyPeer{
+				{
+					// Allow access towards outside the cluster for the virtual machines (guest nodes),
+					// But deny access to other cluster's pods and services with the exceptions below
+					// IPv4
+					IPBlock: &networkingv1.IPBlock{
+						CIDR:   "0.0.0.0/0",
```

Comment:
There is a netip.IPv4Unspecified 

https://pkg.go.dev/net/netip#IPv4Unspecified

Reply:
it returns "0.0.0.0", but i need to provide a CIDR, which is "0.0.0.0/0". I'm not seeing any function that provided "0.0.0.0/0", same for "::/0"

Reply:
updated to:
```
netip.PrefixFrom(netip.IPv4Unspecified(), 0).String()
```